app/models/chapter.rb

```diff
@@ -18,7 +18,6 @@ class Chapter < ActiveRecord::Base
   def self.available_to_user(user)
     return Chapter.all if user.has_role?(:organiser) || user.has_role?(:admin) || user.has_role?(:organiser, Chapter)
     return Chapter.find_roles(:organiser, user).map(&:resource)
-    []
```

Comment:
This can't be reached as the line before returns

Reply:
yep, makes sense